### PR TITLE
Updated Optimizely Readme file

### PR DESCRIPTION
**Pull request by trishahanlon**
### Description

Updated the Readme markdown file for Optimizely's example

### Type of Change

- [ ] New Example
- [ ] Example updates (Bug fixes, new features, etc.)
- [x] Other (changes to the codebase, but not to examples)

### New Example Checklist

- [ ] 🛫 `npm run new-example` was used to create the example
- [ ] 📚 The template wasn't used but I carefuly read the [Adding a new example](https://github.com/vercel/examples#adding-a-new-example) steps and implemented them in the example
- [ ] 📱 Is it responsive? Are mobile and tablets considered?

### CODE REVIEW

File: edge-functions/feature-flag-optimizely/README.md [hunk 1]
```diff
@@ -55,42 +55,42 @@ yarn
...
+You can use Optimizely feature flags at the edge with [NextJS](https://nextjs.org/) applications deployed on [Vercel](https://vercel.com/). This example uses Optimizely [Javascript SDK](https://docs.developers.optimizely.com/full-stack/v4.0/docs/javascript-node) inside a [Vercel Edge Function](https://vercel.com/docs/concepts/functions/edge-functions) as a starting point for you to implement experimentation and feature flagging for your experiences at the edge. For a guide to getting started with our platform more generally, this can be combined with the steps outlined in our Javascript Quickstart [here](https://docs.developers.optimizely.com/full-stack/v4.0/docs/javascript-node).
```
Comment (mikechu-optimizely) [inline edge-functions/feature-flag-optimizely/README.md:66]: NIT: Should we link on `[Javascript Quickstart]` instead of `[here]` (if it matters)?

Comment (trishahanlon) [reply edge-functions/feature-flag-optimizely/README.md]: Updated! Thanks!

**Review by leerob — APPROVED** [review]

File: edge-functions/feature-flag-optimizely/README.md [hunk 2]
```diff
@@ -55,42 +55,42 @@ yarn
 yarn dev
 ```
 
-Deploy it to the cloud with [Vercel](https://vercel.com/new?utm_source=github&utm_medium=readme&utm_campaign=edge-middleware-eap) ([Documentation](https://nextjs.org/docs/deployment)).
+Deploy it to the cloud with [Vercel](https://vercel.com/new?utm_source=github&utm_medium=readme&utm_campaign=edge-middleware-eap) and [Next.js documentation](https://nextjs.org/docs/deployment).
```
Comment (lfades) [inline edge-functions/feature-flag-optimizely/README.md:58]: Lets undo this one

Comment (lfades) [inline edge-functions/feature-flag-optimizely/README.md:58]:
```suggestion
Deploy it to the cloud with [Vercel](https://vercel.com/new?utm_source=github&utm_medium=readme&utm_campaign=edge-middleware-eap) ([Documentation](https://nextjs.org/docs/deployment)).
```

**Review by lfades — APPROVED** [review]
Thank you!

**Pull request merged**